provider/app.py

```diff
@@ -157,6 +157,14 @@ def main():
     streamHandler.setFormatter(formatter)
     logger.addHandler(streamHandler)
 
+    # also log to file if /logs is present
+    if os.path.isdir('/logs'):
+        fh = logging.FileHandler('/logs/kafkatriggers.log')
```

Comment:
I believe @bjustin-ibm is right.  You never lose old logs when stopping and restarting the docker containers in whisk.  It grabs the logs from whisk.logs.dir (specified in whisk.properties) and populates the logs within the containers with the content from the logs outside of the containers.